views/default/object/elements/imprint/contents.php

```diff
@@ -29,3 +31,10 @@
 echo elgg_view('object/elements/imprint/byline', $vars);
 echo elgg_view('object/elements/imprint/time', $vars);
 echo elgg_view('object/elements/imprint/access', $vars);
+
+$imprint = elgg_extract('imprint', $vars);
+if (!empty($imprint)) {
+	foreach ($imprint as $item) {
+		echo elgg_view('object/elements/imprint/element', $item);
```

Comment:
this usage does not match the example array (key is not used)... is that intended

Reply:
That view doesn't use keys. It's `['icon_name' => '', 'content' => '']`

Reply:
my bad... i see now how it is used